deepfence_worker/ingesters/common.go

```diff
@@ -156,3 +167,17 @@ func ToMap[T any](data T) map[string]interface{} {
 	_ = json.Unmarshal(out, &bb)
 	return bb
 }
+
+func anyCompleted(data []map[string]interface{}) bool {
+
+	complete := false
+
+	for i := range data {
+		if data[i]["scan_status"].(string) == utils.SCAN_STATUS_SUCCESS {
```

Comment:
Do we need to check if string conversion gave no error? Otherwise, might run into runtime error

Reply:
we dont need to check here as this field is guaranteed to have string data